pkg/genericapiserver/config.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/golang/glog"
 	"gopkg.in/natefinch/lumberjack.v2"
 
+	"k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen/generators/common"
```

Comment:
Eventually I think the generator and generic apiserver will not live in the same repository, so I think this import path shouldn't go this direction. I'm OK with it temporarily (and possibly past me even suggested this?) but I think this definition package should actually be a subdirectory of genericapiserver.

So the import chain should eventually be [specific apiserver] -> generator -> genericapiserver.